commercial/app/model/merchandise/Merchandise.scala

```diff
@@ -89,26 +83,18 @@ case class TravelOffer(id: String,
   }
 }
 
-sealed trait Gender {
-  override def toString: String
-}
-case object Woman extends Gender {
-  override def toString = "Woman"
-}
-case object Man extends Gender {
-  override def toString = "Man"
-}
-
-case class Member(username: String, gender: Gender, age: Int, profilePhoto: String, location: String) extends Merchandise {
+case class Member(username: String,
+                  gender: Gender,
+                  age: Int,
+                  profilePhoto: String,
+                  location: String) extends Merchandise {
 
   val profileId: Option[String] = profilePhoto match {
     case Member.IdPattern(id) => Some(id)
     case _ => None
   }
 
-  val profileUrl: String = profileId.map(id => s"https://soulmates.theguardian.com/landing/$id")
-    .getOrElse("http://soulmates.theguardian.com/")
-
+  val profileUrl: String = s"https://soulmates.theguardian.com/" + ( profileId.map(id => s"landing/$id") getOrElse "" )
```

Comment:
Yep - spotted that one but have just noticed that a bunch of files have stopped compiling after my merge so need to fix that first :(